pkg/apis/postgres-operator.crunchydata.com/v1beta1/standalone_pgadmin_types.go

```diff
@@ -113,10 +118,21 @@ type ServerGroup struct {
 	PostgresClusterSelector metav1.LabelSelector `json:"postgresClusterSelector"`
 }
 
+type PGAdminUser struct {
+	// Whether the user has admin privileges or not.
+	// +optional
+	IsAdmin bool `json:"isAdmin,omitempty"`
```

Comment:
⛏️ I'd prefer this as an enum. In the UI, this [appears](https://www.pgadmin.org/docs/pgadmin4/8.5/user_management.html) as **Role** of _User_ or _Administrator_. That matches the underlying model [here](https://github.com/pgadmin-org/pgadmin4/blob/e61214047333dadd2c1bd0023997600095667ecc/web/migrations/versions/09d53fca90c7_.py#L131-L137).

Kubernetes [conventions](https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#primitive-types) also warn against bool. That page also recommends using `Xyz` rather than `IsXyz`. 🤔 Do we have any other cases of bool in our specs?

---

I like the following structure but didn't feel confident writing the description. Their docs kinda dance around the word "role."

```suggestion
	// The role...
	// Defaults to User. Valid options are Administrator and User.
	// +kubebuilder:validation:Enum={Administrator,User}
	// +optional
	Role string `json:"role,omitempty"`
```

📝 I see that `setup.py` doesn't think about it this way. I don't have any preference for how deep this ☝🏻 arrangement goes. Our code can convert this to a boolean right after reading the spec. 👍🏻